packages/internal/src/diagnostics/index.ts

```diff
@@ -0,0 +1,137 @@
+import { Project, ts } from 'ts-morph';
```

Comment:
re. `internal/src/diagnostics/index.ts`

Starting to think about this, do we want this to be in `@gql.tada/internal`? I don't really see how that'd be beneficial because it's not unlikely that we'll split `check` up further (since it currently does _a lot_ e.g. instantiate the project, call `resolveTypeScriptRootDir`, call `load`)

It feels more like a distinct file to add to the CLI. If we add it to `@gql.tada/internal` I'd expect it to be used by GraphQLSP (or other future tools), but I'm not sure if it ever will be in the current shape 😅